cc-snapshot

```diff
@@ -288,6 +293,18 @@ if [ ${LARGE_TAR_CONTINUE,,} != 'yes' ]; then
   exit 0
 fi
 
+FS="ext4"
+[[ -n ${LABEL:-} ]] || {
```

Comment:
I suggest we move this above the `FS=` declaration; it looks like this block trying to detect the disk is related to the FS variable, but it is not.